### PR TITLE
fix(purge-expired): accept a list of pocket-id's

### PR DESCRIPTION
**Pull request by jrgm**
r? - @vbudhram 

There is more than one ID that needs to be preserved, so this makes it possible.

### CODE REVIEW

File: bin/purge_expired_tokens.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ config.set('db.autoUpdateClients', false);
 program
   .version(package.version)
   .option('-c, --config [config]', 'Configuration to use. Ex. dev')
-  .option('-p, --pocket-id <pocketId>', 'Pocket Client Id. These tokens will not be purged.')
+  .option('-p, --pocket-id <pocketId>', 'Pocket Client Ids. These tokens will not be purged. (CSV)')
```
Comment (vbudhram) [inline bin/purge_expired_tokens.js]: nit: Could update example command line in https://github.com/mozilla/fxa-oauth-server/pull/419/files#diff-ad4b135c187bb7e96c8db4aa28dfa456L14

File: lib/db/mysql/index.js [hunk 2]
```diff
@@ -518,13 +518,23 @@ MysqlStore.prototype = {
       throw new Error('empty ignoreClientId');
     }
 
-    return self.getClient(ignoreClientId)
-      .then(function (ignoreClient) {
+    if (! Array.isArray(ignoreClientId)) {
+      ignoreClientId = [ ignoreClientId ];
+    }
+
+    var clientIds = ignoreClientId.map(function (id) {
```
Comment (vbudhram) [inline lib/db/mysql/index.js]: Not sure how much ES6 you would like, but you could use arrow functions here.

**Comment by vbudhram** [comment]
@jrgm Didn't know that there were multiple pocket ids. This addition looks good to me 👍, r+, merge at will!

File: lib/db/mysql/index.js (hunk 2 above)
Comment (jrgm) [inline lib/db/mysql/index.js]: ... if we ran on es6, but not yet.

**Pull request merged**